theoplayer/how-to-guides/ios/millicast/getting-started.mdx

```diff
@@ -0,0 +1,58 @@
+# Getting started with Millicast for iOS
+
+import Intro from '../../shared/millicast/_getting-started-intro.mdx';
+
+<Intro />
+
+## Usage
+
+1. Follow [our Getting Started guide](../../../getting-started/01-sdks/03-ios/00-getting-started.md)
+   to set up THEOplayer in your iOS app.
+2. Add the `THEOplayer-Integration-Millicast` dependency to your project's Podfile.
+3. Add the `MillicastIntegration` to the player.
+4. Add a `MillicastSource` to your player's source.
+
+### Add the `integration-millicast` dependency
+
+Add the Millicast integration to your project's `Podfile`, as demonstrated below:
+
+```
+pod 'THEOplayer-Integration-Millicast'
+```
```

Comment:
May be also add an `or` for consuming SPM dependency ?

Reply:
Good idea! You can steal [this snippet](https://github.com/THEOplayer/documentation/blob/ae2164274796991a5606daf867f177ad0d60ee58/theoads/getting-started/00-getting-started-ios.mdx?plain=1#L22-L37) from our THEOads iOS Getting Started guide (although that particular guide [has been removed for now](https://github.com/THEOplayer/documentation/pull/189/commits/e2940c30e5b6875f5ed75bedd6532212dbd2c53c)).